ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSourceConfiguration.h

```diff
@@ -22,13 +22,17 @@
  @param componentProvider See @protocol(CKComponentProvider)
  @param context Passed to methods exposed by @protocol(CKComponentProvider).
  @param sizeRange Used for the root layout.
+ @param workThread The optional thread used by the data source to perform its work; this thread must be executing.
  */
 - (instancetype)initWithComponentProvider:(Class<CKComponentProvider>)componentProvider
                                   context:(id<NSObject>)context
-                                sizeRange:(const CKSizeRange &)sizeRange;
+                                sizeRange:(const CKSizeRange &)sizeRange
+                               workThread:(NSThread *)workThread;
```

Comment:
It feels a bit wrong adding this here. I think it may be better to add this to another initializer in a more clearly marked internal header.